### DISCUSSION TITLE
Exclusão Organização / Exclusion Organization

### DISCUSSION THREAD

**Discussion by cl-provecto** [community/community#57340] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Olá, gostaria de uma informação.

Antes eu tinha uma organização com um determinado nome e por algum motivo, que não me recordo agora, precisei excluir a organização.
Hoje tentei criar a organização com o mesmo nome e o Github não deixa e me informa que o nome não está disponivel.

Após a exclusão do nome eu não posso usar novamente, ou existe algum período após a exclusão para que eu possa usar novamente?

Desde já agradeço a atenção.

===== Translate English =====
Hello, I would like some information.

Before, I had an organization with a certain name and for some reason, which I don't remember now, I had to delete the organization.
Today I tried to create the organization with the same name and Github won't let me and informs me that the name is not available.

After the deletion of the name I cannot use it again, or is there a period after the deletion so that I can use it again?

I thank the attention.

**Comment by NyanKaungSet** [comment]
Hello @prove-cto,

According to this [GitHub Docs](https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account), you have to wait 90 days to use the same name again.

> Deleting your organization account removes all repositories, forks of private repositories, wikis, issues, pull requests, and project or organization pages. Your billing will end and, **after 90 days, the organization name becomes available for use on a new user or organization account.**

If you can't still use it again after 90 days, try contacting the [GitHub Support](https://support.github.com/).

Hope this helps,
